pybamm/spatial_methods/finite_volume.py

```diff
@@ -146,11 +146,12 @@ def divergence(self, symbol, discretised_symbol, boundary_conditions):
                 out = divergence_matrix @ ((r_edges**2) * discretised_symbol)
             elif submesh.coord_sys == "cylindrical polar":
                 out = divergence_matrix @ (r_edges * discretised_symbol)
-        else:
+        elif submesh.coord_sys == "cartesian":
             out = divergence_matrix @ discretised_symbol
-
         return out
```

Comment:
I think this should have an else to catch potential issues in the future. The else could raise an error for an unknown type. I know this should be caught by
```python
        # check coord_sys against KNOWN_COORD_SYS
        if coord_sys is not None and coord_sys not in KNOWN_COORD_SYS:
            raise ValueError(
                f"Coordinate system is {coord_sys}, not in"
                " cartesian, cylindrical polar, spherical polar, None")
```
but it is good to write code as safely as possible.

Reply:
Sorry for the disappear. Done now.